__tests__/app.test.js

```diff
@@ -55,4 +55,49 @@ describe("/api/article/:article_id", () => {
         });
     });
   });
+
+  describe("PATCH", () => {
+    test("200: Returns the updated file", () => {
+      const output = convertTimestampToDate({ ...data.articleData[2], article_id: 3 });
+      output.created_at = output.created_at.toJSON();
+      output.votes -= 10;
+      return request(app)
+        .patch("/api/articles/3")
+        .send({ inc_votes: -10 })
+        .expect(200)
+        .then((res) => {
+          expect(res.body.article).toEqual(output);
```

Comment:
You can get away with just checking if the votes was actually decreased. Something like `expect(res.body.article.votes).toBe(res.body.article.votes - 10)` in this example. You might want to store it in a variable though!

Reply:
That is very true, the test is very much based on the similar test for the GET path so that is why it checks everything.